libraries/src/HTML/HTMLHelper.php

```diff
@@ -1295,7 +1295,13 @@ protected static function addFileToBuffer($path = '', $ext = '', $debugMode = fa
 	 */
 	protected static function convertToRelativePath($path)
 	{
-		$relativeFilePath = Uri::root(true) . str_replace(JPATH_ROOT, '', $path);
+		// Remove JPATH_ROOT only if it is at beginning of full path
+		if (strpos($path, JPATH_ROOT) === 0)
+		{
+			$path = substr_replace($path, '', 0, strlen(JPATH_ROOT));
+		}
+
```

Comment:
```suggestion
 			$path = strpos($path, JPATH_ROOT) === 0 ? substr_replace($path, '', 0, strlen(JPATH_ROOT)) : $path;
```

You can write it as a one line (using the elvis operator)

Reply:
A lot cooler this way!

Reply:
@dgrammatiko I's a normal ternary operator, not Elvis. Elvis would be `?:`.

Reply:
Yes, that was a ternary...